doc/command_syntax_usage.rst

```diff
@@ -38,6 +38,20 @@ over SSH. Each authentication identity requires its own auth entry.
 
 *Note:* --password not being passed or passed as empty are considered the same thing.
 
+---------------------------
+Password-protected SSH key
```

Comment:
I've heard this referred to more often as **SSH Key with a Passphrase**. But I'm flexible if you have heard it differently.

Reply:
Changed to your title; thank you!